app/views/spree/admin/orders/index.html.haml

```diff
@@ -41,7 +41,7 @@
             %span{'ng-show' => "sorting == 'completed_at asc'"}= "&#x25B2;".html_safe
             %span{'ng-show' => "sorting == 'completed_at desc' || sorting === undefined"}= "&#x25BC;".html_safe
 
-      - ['number', 'state', 'payment_state', 'shipment_state', 'email', 'total'].each do |column_name|
+      - ['number', 'state', 'payment_state', 'shipment_state', 'email', 'bill_address_lastname', 'total'].each do |column_name|
```

Comment:
Why is it called `lastname` here and full_name later?

Reply:
Actually we sort the orders with `bill_address.lastname` but we display in the column the `bill_address.fullname` (ie. `"#{firstname} #{lastname}".strip`)

Reply:
Okay, that's confusing, but I see how that can be the desired behaviour.